File: WebApplication/src/modules/annual-administartive-expenses/dtos/annual-administrative-expenses-history-search-result.dto.ts
import { AnnualAdministrativeExpensesHistoryDto } from "./annual-administrative-expenses-history.dto";

export class AnnualAdministrativeExpensesHistorySearchResultDto {
  authorityName: string;
  periodName: string;
  histories: AnnualAdministrativeExpensesHistoryDto[];
}

File: Zopoesht.Application/AdministrativeExpenses/Dtos/AnnualAdministrativeExpensesHistorySearchResultDto.cs
namespace Zopoesht.Application.AdministrativeExpenses.Dtos
{
    public class AnnualAdministrativeExpensesHistorySearchResultDto
    {
        public string AuthorityName { get; set; }

        public string PeriodName { get; set; }

        public List<AnnualAdministrativeExpensesHistoryDto> Histories { get; set; }
    }
}
